### PR TITLE
Fix hgvs parameter

### PR DESCRIPTION
**Pull request by susannasiebert**
This supersedes PR #171. It implements option B of #170.

### CODE REVIEW

File: detect_variants/vep.cwl [hunk 1]
```diff
@@ -25,6 +25,8 @@ inputs:
         inputBinding:
             prefix: "-i"
             position: 1
+    reference:
+        type: File
```
Comment (tmooney) [inline detect_variants/vep.cwl]: Is the reference only required when `--hgvs` is specified?  If so, should this be optional (or possibly a [dependent parameter](http://www.commonwl.org/v1.0/UserGuide.html#Record_inputs,_dependent_and_mutually_exclusive_parameters))?

Comment (jasonwalker80) [inline detect_variants/vep.cwl]: Should the reference also be optional? Is it only needed when the `--hgvs` option is set to true in this API version (86)?

Comment (susannasiebert) [inline detect_variants/vep.cwl]: I made the reference optional. However, with that change if someone sets `hgvs` to `true` and doesn't specify a reference, the workflow will still dispatch but VEP will fail. 

I tried to implement Tom's suggestion of dependent_parameters but it seems like you can't make the dependent parameter set optional. I tried a few things with Tom's help but couldn't get it to work.

**Comment by susannasiebert** [comment]
@tmooney I like this suggestion. @jasonwalker80 what do you think?

**Comment by susannasiebert** [comment]
@tmooney The way we can get around that, I think is to keep the outside hgvs parameter and if it is set to true, set the hgvs_reference for the vep step to the main reference.

**Pull request merged**